influxdb3_write/src/write_buffer/mod.rs

```diff
@@ -77,6 +77,9 @@ pub enum Error {
 
     #[error("walop in file {0} contained data for more than one segment, which is invalid")]
     WalOpForMultipleSegments(String),
+
+    #[error("error: {0}")]
+    Other(#[from] anyhow::Error),
```

Comment:
Is there a reason for us using just a catch all here rather than specific variants?

Reply:
I can change them into specific variants, I wasn't sure what to do here. These errors are ones that actually shouldn't happen, so the receiver can't do anything about it. I was just tossing it into a catch all so that it would at least get logged and get some sort of message back to the user. Previously they were doing a panic, which I wanted to remove. I can add specific thiserror variants if you think this is gross 😄

Reply:
Refactored to have a specific error, incoming...